src/Interpreters/SystemLog.h

```diff
@@ -142,12 +148,6 @@ class SystemLog : public SystemLogBase<LogElement>, private boost::noncopyable,
 
     StoragePtr getStorage() const;
 
-    /** Creates new table if it does not exist.
```

Comment:
Just recently it was `private` and I moved `prepareTable` to `protected` in https://github.com/ClickHouse/ClickHouse/pull/64393
This function is being promoted pretty quickly 😄